### PR TITLE
pop module from stack after luaL_register

### PR DESCRIPTION
**Pull request by kroggen**
It was lacking to add `lua_pop()` on 2 places

### CODE REVIEW

**Review by hayarobi — APPROVED** [review]
lgtm

File: contract/db_module.c
Comment (hayarobi) [inline contract/db_module.c:1]: It changes init positon of static variables. I'm too past to follow C code well. What is different?

Comment (kroggen) [reply contract/db_module.c:1]: On all other modules these static variables are outside the function that use them, so I made this file have the same structure

Comment (kroggen) [reply contract/db_module.c:1]: [here](https://github.com/aergoio/aergo/blob/develop/contract/system_module.c#L459) is an example

**Pull request merged**